pkg/arvo/app/spider.hoon

```diff
@@ -18,18 +18,27 @@
   $:  starting=(map yarn [=trying =vase])
       running=trie
       tid=(map tid yarn)
-      serving=(map tid [@ta =mark =desk])
+      serving=(map tid [(unit @ta) =mark =desk])
   ==
 ::
 +$  clean-slate-any
   $^  clean-slate-ket
   $%  clean-slate-sig
       clean-slate-1
       clean-slate-2
+      clean-slate-3
       clean-slate
   ==
 ::
 +$  clean-slate
+  $:  %4
+      starting=(map yarn [=trying =vase])
+      running=(list yarn)
+      tid=(map tid yarn)
+      serving=(map tid [(unit @ta) =mark =desk])
```

Comment:
IMO, given that we're serious about multi-desk support for spider, `serving` should probably be just `(map tid [eyre-id=@ta =mark])` and we should store the desk of the thread alongside the thread itself. Defaulting to `%noun` as a mark feels kinda wrong tbh.